cmd/start/main.go

```diff
@@ -432,3 +437,38 @@ func getCommitSHA(dir string) (string, error) {
 	}
 	return strings.TrimSpace(string(content)), nil
 }
+
+func gitLfsInUse(logger logging.LeveledLoggerInterface, dir string) bool {
+	stdout, stderr, err := command.RunInDir("git", []string{
+		"lfs",
+		"ls-files",
+		"--all",
+	}, dir)
+	if err != nil {
+		logger.Errorf(string(stderr))
+		log.Fatal(err)
+	}
+
+	// no error, so count lines/files
+	files := 0
+	endOfLine := '\n'
+	for _, c := range string(stdout) {
+		if c == endOfLine {
+			files++
+		}
+	}
+
+	return files > 0
+}
+
+func gitLfsPullFiles(logger logging.LeveledLoggerInterface, dir string) {
+	stdout, stderr, err := command.RunInDir("git", []string{
+		"lfs",
+		"pull",
+	}, dir)
+	if err != nil {
+		logger.Errorf(string(stderr))
+		log.Fatal(err)
```

Comment:
I'd rather return the error and handle in the caller. I'd wrap the original error, and pass it back together with `stderr`.